src/docs/asciidoc/getting-started.adoc

```diff
@@ -27,33 +27,46 @@ It has transitive dependencies on:
 * Netty v4.1.x
 
 [[getting-started-understanding-bom]]
-== Understanding the BOM
+== Understanding the BOM and versioning scheme
 
 `Reactor Netty` is part of the `Project Reactor BOM` (since the `Aluminium` release train).
 This curated list groups artifacts that are meant to work well together, providing
 the relevant versions despite potentially divergent versioning schemes in these artifacts.
 
-The `BOM` (Bill of Materials) is itself versioned, using a release train scheme
-with a codename followed by a qualifier. The following list shows a few examples:
+Note the versioning scheme has changed between 0.9.x and 1.0.x (Dysprosium and Europium).
 
-[verse]
-Aluminium-RELEASE
-Californium-BUILD-SNAPSHOT
-Aluminium-SR1
-Bismuth-RELEASE
-Californium-SR32
+Artifacts follow a versioning scheme of `MAJOR.MINOR.PATCH-QUALIFIER` while the BOM is versioned using a CalVer inspired scheme of `YYYY.MINOR.PATCH-QUALIFIER`, where:
 
-The codenames represent what would traditionally be the `MAJOR.MINOR` number. They (mostly)
+ * `MAJOR` is the current generation of Reactor, where each new generation can bring fundamental changes to the structure of the project (which might imply a more significant migration effort)
+ * `YYYY` is the year of the first GA release in a given release cycle (like 3.4.0 for 3.4.x)
```

Comment:
let's use Reactor Netty version as an example and not Reactor Core i.e.

Reply:
yeah good idea